lib/docs/src/tag.stories.tsx

```diff
@@ -1,24 +1,54 @@
 import { Meta } from "@storybook/react";
 import { Fragment } from "react";
-import { DivPx, Tag } from "../../core/src";
+import { Button, DivPx, Tag } from "../../core/src";
+import { Utils } from "./utils";
 
-export default {
-	title: "Draft/Tag",
+const colors: string[] = [
+	"red",
+	"yellow",
+	"green",
+	"blue",
+	"indigo",
+	"purple",
+	"pink",
+	"gray",
+];
+
+const meta: Meta = {
+	title: "Components/Tag",
 	component: Tag,
-} as Meta;
-
-export const Primary = (): JSX.Element => {
-	const colors: string[] = [
-		"red",
-		"yellow",
-		"green",
-		"blue",
-		"indigo",
-		"purple",
-		"pink",
-		"gray",
-	];
+	argTypes: {
+		children: Utils.arg("text"),
+		color: Utils.arg(colors),
+	},
+};
+
+Utils.page.component(meta, { sticky: true, shots: [] });
```

Comment:
The "shots" option here should be used to show a gallery of tags. Find the component below and put it here

<img width="339" alt="image" src="https://user-images.githubusercontent.com/5953369/119462308-c721db80-bd6a-11eb-912f-22d67d56c919.png">